src/tasks.ts

```diff
@@ -40,10 +40,28 @@ export const reportResults = async (result: Result): Promise<void> => {
 
 export const runTask = async (task: Commands): Promise<void> => {
   switch (task) {
-    case Commands.CreateGitIgnoreBackup:
-      return runCommand("sed -i.bak -e '/dist/d' .gitignore");
     case Commands.RemoveNpmrc:
+      /**
+       * Remove .npmrc file from the repository during release where
+       * @ridedott/semantic-release-action action is used.
+       */
+
       return runCommand('rm -rf .npmrc');
```

Comment:
`-r` means recursive, it's needed only for directories. We should leave `-f` tho for cases where the file does not exist.

Reply:
👍🏼

Reply:
I think you did the opposite?